IntroOOP/BankAccount/BankAccount.cs

```diff
@@ -114,18 +113,44 @@ public void WithdrawMoney(decimal value)
                 Console.WriteLine($"Текущий баланс счета: {Balance}");
             }
         }
-        
+
     }
 
     /// <summary>
-    /// Метод печатает состояние
+    /// Перевод денег с одного счета на другой
     /// </summary>
-    public void GetInfo() 
+    /// <param name="source">Счет-донор</param>
+    /// <param name="amount">Сумма перевода</param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException">Проверка</exception>
+    public bool TransferMoney(BankAccount source, decimal amount)
     {
-        Console.WriteLine("Номер счета: {0}\nБаланс: {1}\nТип: {2}", ID, Balance, Type);
-    }
+        if (amount < 0)
+        {
+            throw new ArgumentException("Размер перевода не может быть отрицательным!");
+        }
+        else
+        {
+            if (source.Balance < amount)
+            {
+                Console.WriteLine($"Перевод на сумму {amount} RUB не выполнен!\nНа счете №{source.ID} недостаточно средств!\n");
+                return default;
```

Comment:
```suggestion
                return false;
```
Не надо так. Если тип bool, то не надо шокировать публику нестандартными значениями.